packages/contracts/tasks/deploy/maci/09-voiceCreditProxyFactory.ts

```diff
@@ -0,0 +1,109 @@
+import type {
+  ConstantInitialVoiceCreditProxyFactory,
+  ERC20VotesInitialVoiceCreditProxyFactory,
+} from "../../../typechain-types";
+
+import { info, logGreen } from "../../../ts/logger";
+import { EDeploySteps } from "../../helpers/constants";
+import { ContractStorage } from "../../helpers/ContractStorage";
+import { Deployment } from "../../helpers/Deployment";
+import {
+  EContracts,
+  EInitialVoiceCreditProxies,
+  EInitialVoiceCreditProxiesFactories,
+  IDeployParams,
+} from "../../helpers/types";
+
+const deployment = Deployment.getInstance();
+const storage = ContractStorage.getInstance();
+
+/**
+ * Deploy step registration and task itself
+ */
+deployment
+  .deployTask(EDeploySteps.InitialVoiceCreditProxyFactory, "Deploy initial voice credit proxy factory")
+  .then((task) =>
+    task.setAction(async ({ incremental }: IDeployParams, hre) => {
+      deployment.setHre(hre);
+      const deployer = await deployment.getDeployer();
+
+      const { deployConstantInitialVoiceCreditProxyFactory, deployERC20VotesInitialVoiceCreditProxyFactory } =
+        await import("../../../ts/deploy");
+
+      const voiceCreditProxyType = deployment.getDeployConfigField(EContracts.Poll, "initialVoiceCreditProxy");
+
+      const needDeploy = deployment.getDeployConfigField(voiceCreditProxyType as EInitialVoiceCreditProxies, "deploy");
+
+      if (needDeploy === false) {
+        return;
+      }
+
+      const skipDeployConstantInitialVoiceCreditProxy = voiceCreditProxyType !== EInitialVoiceCreditProxies.Constant;
+      const skipDeployERC20VotesInitialVoiceCreditProxy =
+        voiceCreditProxyType !== EInitialVoiceCreditProxies.ERC20Votes;
+
+      if (!skipDeployConstantInitialVoiceCreditProxy) {
+        const constantInitialVoiceCreditProxyContractAddress = storage.getAddress(
+          EInitialVoiceCreditProxies.Constant,
+          hre.network.name,
+        );
+
+        if (incremental && constantInitialVoiceCreditProxyContractAddress) {
+          // eslint-disable-next-line no-console
```

Comment:
This can be removed

Reply:
I mean you need to remove only `// eslint-disable-next-line no-console` because there is no console